packages/react/src/components/Pagination/Pagination.js

```diff
@@ -162,7 +162,7 @@ export default class Pagination extends Component {
     return !pageSizesChanged && !pageChanged && !pageSizeChanged
       ? null
       : {
-          page: pageSizeChanged && 1 || pageChanged && page || currentPage,
+          page: (pageSizeChanged && 1) || (pageChanged && page) || currentPage,
```

Comment:
intentional?

Reply:
This is also a weird one ... When I reset the branch and pulled I got a whole bunch of unrelated changes ... definitely not intentional

Reply:
Though it seems like this is the same as `master` right now ... https://github.com/carbon-design-system/carbon/blob/master/packages/react/src/components/Pagination/Pagination.js#L165